web-demos/custom-login-button.html

```diff
@@ -2,7 +2,7 @@
 <html>
 
 <head>
-  <title>Pryv - Javascript Lib</title>
```

Comment:
This is not yet working but I suppose you are working on it?

Reply:
ack

Reply:
@IevaNavikiene check it out, it should be fixed

Reply:
fixed :)